src/handler/GenerateProjectHandler.ts

```diff
@@ -132,6 +155,16 @@ async function specifyArtifactId(): Promise<string> {
     });
 }
 
+async function specifyName(): Promise<string> {
+    const defaultName: string = vscode.workspace.getConfiguration("spring.initializr").get<string>("defaultName");
+    return await getFromInputBox({
+        placeHolder: "e.g. demo (will turn into DemoApplication class - mostly the same as ArtifactId)",
+        prompt: "Input project name.",
+        validateInput: nameValidation,
+        value: defaultName,

```

Comment:
As you mentioned: 
>  mostly the same as ArtifactId

Let's remove this step, just re-use the value of artifactId.

Reply:
Done.